File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency2/model/Dto/FlightDto.cs
namespace model.Dto
{
    public class FlightDto : Entity<long>
    {

        public long startId { get; set; }
        public long destinationId { get; set; }
        public DateTime startDate { get; set; }
        public int nrOfSeats { get; set; }

        public FlightDto(long id, long startId, long destinationId, DateTime startDate, int nrOfSeats) : base(id)
        {
            this.startId = startId;
            this.destinationId = destinationId;
            this.startDate = startDate;
            this.nrOfSeats = nrOfSeats;
        }


    }
}

File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency/Model/src/main/java/domain/dto/FlightDto.java
package domain.dto;


import domain.Entity;

import java.time.LocalDateTime;

public class FlightDto extends Entity<Long> {
    private final long startId;
    private final long destinationId;
    private final LocalDateTime startDate;
    private final int nrOfSeats;

    public FlightDto(long startId, long destinationId, LocalDateTime startDate, int nrOfSeats) {
        this.startId = startId;
        this.destinationId = destinationId;
        this.startDate = startDate;
        this.nrOfSeats = nrOfSeats;
    }

    public long getStartId() {
        return startId;
    }

    public long getDestinationId() {
        return destinationId;
    }

    public LocalDateTime getStartDate() {
        return startDate;
    }

    public int getNrOfSeats() {
        return nrOfSeats;
    }
}

